### PR TITLE
NMS-13810: Phase 4 flows documentation: Nephron for aggregation

### PR DESCRIPTION
**Pull request by Bonrob2**
First draft. I am not sure where this content from the Core guide (https://docs.opennms.com/meridian/2021/operation/flows/aggregation.html) should go. Also would appreciate suggestions for Nephron troubleshooting topics/content.

### CODE REVIEW

File: docs/antora.yml [hunk 1]
```diff
@@ -10,5 +10,8 @@ asciidoc:
     xrefstyle: short@
     compatible-horizon: '26.1.0-SNAPSHOT'
```
Comment (indigo423) [inline docs/antora.yml:11]: I think we shouldn't have SNAPSHOT version here, right? Is Horizion 26.1.0 the minimal version we need?

**Review by indigo423 — APPROVED** [review]
Besides the compatibility SNAPSHOT version, looks good to me.

File: docs/antora.yml (hunk 1 above)
Comment (Bonrob2) [reply docs/antora.yml:11]: Good catch! I took the requirements content from the Nephron Field Guide on Confluence, and it says Horizon 26/Meridian 2020. @swachter , can you confirm?

**Comment by Bonrob2** [comment]
@indigo423 , updated but requires an approval.

**Review by indigo423 — APPROVED** [review]
LGTM!

**Pull request merged**